packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1562,21 +1570,74 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {undefined}
    * @public
    * @memberof Meeting
+   * @deprecated after v1.89.3
    */
   closeRemoteStream() {
-    return Media.stopStream(this.mediaProperties.remoteStream).then(() => {
+    LoggerProxy.logger.warn('Meeting:index#closeRemoteStream --> [DEPRECATION WARNING]: closeRemoteStream has been deprecated after v1.89.3');
+    this.closeRemoteTracks();
+  }
+
+  /**
+   * Removes the remote tracks on the class instance and triggers an event
+   * to developers
+   * @returns {undefined}
+   * @memberof Meeting
+   */
+  closeRemoteTracks() {
+    const {
+      remoteAudioTrack,
+      remoteVideoTrack,
+      remoteShare
+    } = this.mediaProperties;
+
+    /**
+     * Triggers an event to the developer
+     * @param {string} mediaType Type of media that was stopped
+     * @returns {void}
+     * @inner
+     */
+    const triggerMediaStoppedEvent = (mediaType) => {
       Trigger.trigger(
         this,
         {
           file: 'meeting/index',
-          function: 'closeRemoteStream'
+          function: 'closeRemoteTracks'
         },
         EVENT_TRIGGERS.MEDIA_STOPPED,
         {
-          type: EVENT_TYPES.REMOTE
+          type: mediaType
         }
       );
-    });
+    };
+
+    /**
+     * Stops a media track and emits an event
+     * @param {MediaStreamTrack} track Media track to stop
+     * @param {string} type Media track type
+     * @returns {Promise}
+     * @inner
+     */
+    // eslint-disable-next-line arrow-body-style
+    const stopTrack = (track, type) => {
+      return Media.stopTracks(track)
+        .then(() => {
+          const isTrackStopped = track && track.readyState === ENDED;
+          const isWrongReadyState = track && !isTrackStopped;
+
+          if (isTrackStopped) {
+            triggerMediaStoppedEvent(type);
+          }
+          else if (isWrongReadyState) {
+            LoggerProxy.logger.warn(`Meeting:index#closeRemoteTracks --> Error: MediaStreamTrack.readyState is ${track.readyState} for ${type}`);
```

Comment:
When would this get triggred ?

Reply:
This will get triggered if we have a media track but the `readyState` is not equal to the constant `ENDED`. The `else if` is to avoid this getting logged if the function is called but the tracks on `mediaProperties` has already been set to `null`.  

Keep in mind right now `Media.cleanUp` is called at least twice, with it check it you won't get warnings if the tracks have already been cleared.

Reply:
thanks